build.gradle

```diff
@@ -84,7 +77,56 @@ task('exportVersion', description: 'Write the current build version out to a pro
     }
 }
 
-task('release', description: 'Release the project and publish to all repos.', group: 'release', dependsOn: [exportVersion, tagVersion/*, publishToPackageRepo*/])
+task('publishToPackageRepo', type: HttpTask, group: 'publishing') {
+    description 'Publishes the DEB and RPM packages to the package repository'
+    dependsOn configurations.debBundle, configurations.rpmBundle, buildAll
+    mustRunAfter tagVersion
```

Comment:
Why?

Reply:
The `mustRunAfter` instruction, you mean?

It isn't really necessary, but my thinking was that if the `tagVersion` fails it is easier to clean up that this task. So I would rather `tagVersion` run first. In general, I would prefer easy-to-cleanup publication tasks to run before other publication tasks.

Of course, this does add some minimal complication and restrains what Gradle can do task-ordering wise.

I'm happy to remove it.

Reply:
While I get what your saying, it makes the task graph a little less intuitive as well as making this particular publishing task different from all the others. I'd rather they all match.

Reply:
To be clear that doesn't mean make them all depend on `tagVersion` because as far as i'm concerned that's also a publishing task.

Reply:
Got it. I'll remove this line.